### ISSUE TITLE
Generation of 'css/style.css' failed, while sass has been installed

### ISSUE THREAD

**Issue by Neftedollar** [LambdaFactory/Fornax#14]
I've installed sass, but this message still provided and styles are not generated.
```sh
PS C:\Users\neftedollar\source\playground\fornax> fornax new
PS C:\Users\neftedollar\source\playground\fornax> fornax watch
[13:48:19] 'C:\Users\neftedollar\source\playground\fornax\_public\archive.html' generated in 120ms
[13:48:20] 'C:\Users\neftedollar\source\playground\fornax\_public\index.html' generated in 555ms
[13:48:20] Generation of 'css/style.css' failed. Please check you have installed the Sass compiler if you are going to be using files with extension .scss. https://sass-lang.com/install
[13:48:20] Generation of 'css/_settings.css' failed. Please check you have installed the Sass compiler if you are going to be using files with extension .scss. https://sass-lang.com/install
[13:48:21] 'C:\Users\neftedollar\source\playground\fornax\_public\posts/2017-05-30-welcome.html' generated in 553ms
[13:48:21] 'C:\Users\neftedollar\source\playground\fornax\_public\posts/2017-05-31-test-styles.html' generated in 5ms
[13:48:21] Watch mode started. Press any key to exit.
[13:48:21 INF] Smooth! Suave listener started in 57.38ms with binding 127.0.0.1:8080
 Exiting...
PS C:\Users\neftedollar\source\playground\fornax> sass
Compile Sass to CSS.

Usage: sass <input.scss> [output.css]
       sass <input.scss>:<output.css> <input/>:<output/> <dir/>
```

**Comment by Krzysztof-Cieslak** [comment]
Fixed by #15

**Comment by Neftedollar** [comment]
@fsharplang-ru 💪 :)
